2019/bamberg/schedule.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: location_detail
+partner_site: capetown
```

Comment:
This doesn't match the directory it is in. Is this incorrect?

Reply:
addressed in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/ea27995cfeab1f83e8100a17a482be32ba26a1e2